python/mlcroissant/mlcroissant/_src/operation_graph/operations/filter.py

```diff
@@ -23,13 +23,13 @@ def __call__(self, *dirs: Path) -> list[Path]:
         """See class' docstring."""
         includes = fnmatch.translate(self.node.includes)
         includes_re = re.compile(includes)
-        included_files: list[epath.Path] = []
+        included_files: list[Path] = []
```

Comment:
nit: `included_files` could now be renamed to `included_paths`.

(`Path` used to be named `File`. I think there's this discrepancy in several places.)